backend/agentd/session.go

```diff
@@ -212,7 +219,7 @@ func (s *Session) Start() (err error) {
 
 	for _, sub := range s.cfg.Subscriptions {
 		topic := messaging.SubscriptionTopic(org, env, sub)
-		logger.Debugf("Subscribing to topic %q", topic)
+		logger.WithField("topic", topic).Debug("Subscribing to topic")
```

Comment:
for the sake of consistency, can we change the capitalization of Subscribing to subscribing?